src/dlib_export.h

```diff
@@ -0,0 +1,20 @@
+#if !defined(DLIB_EXPORT)
+	#define DLIB_EXPORT /* NOTHING */
```

Comment:
This macro should be named `DLL_EXPORT`
also, did you know that `#ifndef DLL_EXPORT` is a synonym for `#if !defined(DLL_EXPORT)`?